src/content/_index.en.md

```diff
@@ -3,15 +3,16 @@ title: "Submariner"
 ---
 
 # Submariner
-#### Submariner enables direct networking between Pods and Services in different Kubernetes clusters, either on premise or in the cloud.
 
-## Why Submariner?
+**Submariner enables direct networking between Pods and Services in different Kubernetes clusters, either on premise or in the cloud.**
```

Comment:
This actually makes the text smaller, from:

![image](https://user-images.githubusercontent.com/1176434/89518166-d2574b00-d7da-11ea-9fc5-29ff7d63f835.png)

To:

![image](https://user-images.githubusercontent.com/1176434/89518226-e7cc7500-d7da-11ea-9e46-70fe3c24f7fe.png)


May be we could use html for that instead of ** ?

Reply:
Good idea, done in https://github.com/submariner-io/submariner-website/pull/234 (and I'll rebase this).